.github/workflows/build.yml

```diff
@@ -43,7 +43,11 @@ jobs:
         run: make lint
 
       - name: Run tests
-        run: make test
+        run: |
+          curl -L https://get.gear.rs/gear-nightly-linux-x86_64.tar.xz -o - | tar xJ
```

Comment:
Need to rewrite tests using [`Node`](https://docs.gear.rs/gclient/struct.Node.html#method.try_from_path) but not in this PR.